tutorials/mnist_tutorial_jsma.py

```diff
@@ -112,6 +112,14 @@ def mnist_tutorial_jsma(train_start=0, train_end=60000, test_start=0,
     grid_shape = (nb_classes, nb_classes, img_rows, img_cols, channels)
     grid_viz_data = np.zeros(grid_shape, dtype='f')
 
+    # by default, we have softmax after a CNN, remove it here
```

Comment:
It's probably better to change JSMA to take probabilities rather than to modify the tutorial